src/pages/DatasetRegistration.js

```diff
@@ -217,52 +145,45 @@ class DatasetRegistration extends Component {
     const value = e.target.value;
     this.setState(prev => {
       prev.formData[field] = value;
+      prev.disableOkBtn = false;
       return prev;
-    }, () => this.checkValidations());
-  };
-
-  checkValidations() {
-    if (this.state.showValidationMessages === true && this.state.step === 1) {
-      this.verifyStep1();
-    } else if (this.state.showValidationMessages === true && this.state.step === 2) {
-      this.verifyStep2();
-    } else if (this.state.showValidationMessages === true && this.state.step === 3) {
-      this.verifyStep3();
-    }
+    });
   };
 
   handleCheckboxChange = (e) => {
     const field = e.target.name;
     const value = e.target.checked;
     this.setState(prev => {
       prev.formData[field] = value;
-      return prev;
-    }, () => this.checkValidations());
-  };
-
-  step1 = (e) => {
-    this.setState(prev => {
-      prev.step = 1;
+      prev.disableOkBtn = false;
       return prev;
     });
-    window.scrollTo(0, 0);
   };
 
-  step3 = (e) => {
-    this.setState(prev => {
-      prev.step = 3;
-      return prev;
-    });
-    window.scrollTo(0, 0);
+  validateRequiredFields(formData) {
```

Comment:
Manually validates the required fields since the html attribute `required` didn't seem to do anything

Reply:
I could be wrong, but I don't think we're submitting a form here which is why required doesn't work. We're really just manipulating state and calling API actions based on that state. 

https://www.w3schools.com/tags/att_input_required.asp